src/QGOOut.py

```diff
@@ -65,16 +75,16 @@ def retNames(item):
         pre_sol_dfs = self.get_pre_dfs(sol_node_list, sol_edge_list)
 
         # Concatenating all the separate dataframes into one big DataFrame
-        single_prob_df = pd.concat(pre_problem_dfs, ignore_index=True).reset_index(drop=True)
-        single_sol_df = pd.concat(pre_sol_dfs, ignore_index=True).reset_index(drop=True)
+        single_prob_df: pd.DataFrame = pd.concat(pre_problem_dfs, ignore_index=True).reset_index(drop=True)
+        single_sol_df: pd.DataFrame = pd.concat(pre_sol_dfs, ignore_index=True).reset_index(drop=True)
 
         # Finally, generating the actual GeoDataFrame that can be manipulated
         geo_prob_df = gpd.GeoDataFrame(single_prob_df, geometry='geometry', crs='epsg:4326')
         geo_sol_df = gpd.GeoDataFrame(single_sol_df, geometry='geometry', crs='epsg:4326')
         
         # Output to file
-        geo_prob_df.to_file('/home/OpenQGO/data_out/QGOprob.geojson', driver='GeoJSON')  
-        geo_sol_df.to_file('/home/OpenQGO/data_out/QGOsol.geojson', driver='GeoJSON')  
+        geo_prob_df.to_file(DATA_OUT_DIR / 'QGOprob.geojson', driver='GeoJSON')  
+        geo_sol_df.to_file(DATA_OUT_DIR / 'QGOsol.geojson', driver='GeoJSON')  
```

Comment:
This makes the file paths relative, so they'll work even outside of the docker container.